### PR TITLE
Remove deprecated serviceAccount

### PR DESCRIPTION
**Pull request by fengshunli**
### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/fluid/v0.7.0/templates/csi/daemonset.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ spec:
       labels:
         app: csi-nodeplugin-fluid
     spec:
-      serviceAccount: fluid-csi
+      serviceAccountName: fluid-csi
```
Comment (cheyang) [inline charts/fluid/v0.7.0/templates/csi/daemonset.yaml:17]: Please don't update the released version. thanks.

Comment (cheyang) [inline charts/fluid/v0.7.0/templates/csi/daemonset.yaml:17]: Please don't update the released version. thanks.

Comment (fengshunli) [reply charts/fluid/v0.7.0/templates/csi/daemonset.yaml]: done

Comment (fengshunli) [reply charts/fluid/v0.7.0/templates/csi/daemonset.yaml]: done

**Review by cheyang — APPROVED** [review]

**Pull request merged**